site_scons/site_tools/extra/extra.py

```diff
@@ -26,53 +26,49 @@
 import os
 
 from SCons.Builder import Builder
-from SCons.Script import Dir
 from SCons.Script import WhereIs
 
-def supports_custom_format(clang_exe):
+def _supports_custom_format(clang_exe):
     """Get the version of clang-format"""
 
     try:
         rawbytes = subprocess.check_output([clang_exe, "-version"])
         output = rawbytes.decode('utf-8')
     except subprocess.CalledProcessError:
-        print("Unsupported clang-format for custom style.  "
-              "Using Mozilla style.")
+        print("Unsupported clang-format for custom style.  Using Mozilla style.")
         return False
 
     match = re.search(r"version (\d+)\.", output)
-    if match and int(match.group(1)) >= 7:
+    if match and int(match.group(1)) >= 12:
```

Comment:
Maybe the version supported could be defined in a configuration file, or with a constant at the beginning file with something like this:
`__CLANG_VERSION_MIN__ = 12 `

Reply:
True, will fix for this file is I need to re-push.